src/main/java/sg/ncl/MainController.java

```diff
@@ -76,6 +76,9 @@
 
     private final RestClient restClient = new RestClient();
 
+    // error messages
+    private final String ERR_SERVER_OVERLOAD = "Our server is currently overloaded with requests. Please try again later.";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "ERR_SERVER_OVERLOAD" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00116)